<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>view_prj_hmiplugin</class>
 <widget class="CBaseView" name="view_prj_hmiplugin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>view_prj_hmiplugin</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>HMI插件项目自动创建</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="5" column="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="chkCreateSubDir">
        <property name="text">
         <string>以项目名称创建子目录</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="edtPrjPath"/>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="btnDir">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>项目名称：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="edtPrjName"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>项目目录：</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="btnBuild">
        <property name="text">
         <string>创建VS2010项目(&amp;B)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>项目描述：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="edtPrjDesc"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CBaseView</class>
   <extends>QWidget</extends>
   <header>ssp_baseview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtPrjName</tabstop>
  <tabstop>edtPrjDesc</tabstop>
  <tabstop>edtPrjPath</tabstop>
  <tabstop>btnDir</tabstop>
  <tabstop>chkCreateSubDir</tabstop>
  <tabstop>btnBuild</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
